cmd/deviceservice/rm/rm.go

```diff
@@ -38,9 +37,9 @@ func NewCommand() *cobra.Command {
 
 			deviceID := args[0]
 			respBody, err := client.DeleteItem(deviceID,
-				config.Conf.MetadataService.DeviceServiceByIDRoute,
-				config.Conf.MetadataService.DeviceServiceBySlugNameRoute,
-				config.Conf.MetadataService.Port)
+				"deviceservice/id/",
+				"deviceservice/name/",
+				"48081")
```

Comment:
odd that this call needed a port number

Reply:
Why so? client.DeleteItem requires the port to build the URL